mayastor/src/rebuild.rs

```diff
@@ -60,7 +62,24 @@ impl fmt::Display for RebuildState {
     }
 }
 
-#[derive(Debug)]
+struct TaskResult {
```

Comment:
please add comments to each of the fields as we do (mostly). Using 3 slashes creates a doc comment i.e

```rust
/// this struct is used for ... 
struct  foo {
/// the blk (??) being rebuilt
blk: u64
/// the  id that is used for...??
id: 64
}
```